recipes/natural_language_processing/chatbot/bootc/Containerfile

```diff
@@ -24,19 +24,32 @@ ARG APPIMAGE=quay.io/ai-lab/${RECIPE}:latest
 ARG SERVERIMAGE=quay.io/ai-lab/llamacpp-python:latest
 
 # Add quadlet files to setup system to automatically run AI application on boot
-COPY quadlet/${RECIPE}.kube quadlet/${RECIPE}.yaml quadlet/${RECIPE}.image /usr/share/containers/systemd
+COPY quadlet/${RECIPE}.kube quadlet/${RECIPE}.yaml /usr/share/containers/systemd
 
 # Modify quadlet files to match the server, model and app image
 RUN sed -e "s|SERVERIMAGE|${SERVERIMAGE}|" \
     -e "s|APPIMAGE|${APPIMAGE}|g" \
     -e "s|MODELIMAGE|${MODELIMAGE}|g" \
     -i \
-    /usr/share/containers/systemd/${RECIPE}.image \
     /usr/share/containers/systemd/${RECIPE}.yaml
 
+# Because images are prepulled, no need for .image quadlet
+# COPY quadlet/${RECIPE}.image /usr/share/containers/systemd
+# RUN sed -e "s|SERVERIMAGE|${SERVERIMAGE}|" \
+#    -e "s|APPIMAGE|${APPIMAGE}|g" \
+#    -e "s|MODELIMAGE|${MODELIMAGE}|g" \
+#    -i \
+#    /usr/share/containers/systemd/${RECIPE}.image
+
+# Setup /usr/lib/containers/storage as an additional store for images.
+# Remove once the base images have this set by default.
```

Comment:
the .image file ensures the images exist when restarting the service - if an image is manually deleted somehow then the service won't fail it will show as activating when pulling the image - is this enough of a reason to include it?  Also, it's important if you _don't_ pre-load the images.

Reply:
The problem is a image always attempts to pull, and no way to tell it to check if the image pre-exists.  Without the .image file the chatbot.yaml file should attempt to pull the image if it does not exists.

Since some of the images are protected behind a registry, we would need to populate a auth.json file, which would be a security risk or require users to setup the auth.json file some other way.  Bottom line, if  you prepull the images, then no need for a .image quadlet, as far as I can see.